home-application/pom.xml

```diff
@@ -63,6 +63,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
+<!--        <dependency>-->
```

Comment:
Interesting why we commented it...

Reply:
I don`t know why we need this on this module, and that is why I removed it